src/webui/www/private/scripts/cache.js

```diff
@@ -31,6 +31,7 @@
 if (window.qBittorrent === undefined)
     window.qBittorrent = {};
 
+if (window.qBittorrent.Cache === undefined) {
 window.qBittorrent.Cache = (() => {
```

Comment:
[Nullish coalescing assignment (`??=`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment) can be used instead of `if` guards. Suppored by browsers since 2020 and conforms with qB compatibility table.

I.e.
```js
window.qBittorrent.Cache ??= (() => {
```

Reply:
Thanks, applied.